backend/src/test/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepositoryTest.java

```diff
@@ -34,7 +37,14 @@
   private static final Facility ABC_1 = getFacility(UUID.randomUUID(), ABC);
   private static final Facility ABC_2 = getFacility(UUID.randomUUID(), ABC);
 
-  private DemoOktaRepository _repo = new DemoOktaRepository();
+  private static final AuthorizationProperties MOCK_PROPS =
```

Comment:
very nit: these objects aren't mocks, they're just using fake variables. i would prefer to rename these objects.